product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/1.6-xml-message-enrichment/1.6.2-Modify-payload-by-adding-a-sibling-using-enrich-mediator/readme.md

```diff
@@ -0,0 +1,178 @@
+# 1.6.2 Modify payload by adding a sibling using enrich mediator
+
+## When to use
+The Enrich Mediator can process a message based on a given source configuration and then perform the specified action on the message by using the target configuration. It is often used to do slight modifications to the payload. There are main three target actions as “Replace”, “Child” and “Sibling”. In this scenario we focus on adding a sibling element to the payload. 
+
+
+## Sample use case
+Assume that we have our original payload to be sent as below. 
+
+original payload
+```
```

Comment:
Let's use xml formatting in all the xml documented places

Reply:
Done